grid.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+
+    	<meta charset="utf-8">
+    	<meta name="description" content="Ice Times">
+    	<meta name="keywords" content="Hockey, Ice Hockey, Pick-Up, Stick 'n' Shoot">
+
+		<title>Ice Times</title>
+
+		<link rel="icon" type="image/ico" href="images/hockey.png"/>
+
+		<!-- external CSS link -->
+		<link href='http://fonts.googleapis.com/css?family=Economica:400,700,400italic,700italic' rel='stylesheet' type='text/css'>
+		<link rel="stylesheet" href="css/reset.css">
+		<link rel="stylesheet" href="css/style.css">
+
+	</head>
+	<body>
+
+    <header>
+      <div class="container">
+
+        <div class="navigation">
+          <ul>
+            <li class="dropDown"><a href="grid.html">Rinks</a>
+						<!-- <div class="dropDownInner">
+							<ul>
+								<li><a href="#"> Cabin John Ice Arena </a></li>
+								<li><a href="#"> Kettler Capital's Iceplex </a></li>
+								<li><a href="#"> Rockville Ice Arena </a></li>
+								<li><a href="#"> Skatequest </a></li>
+								<li><a href="#"> Wheaton Ice Rink </a></li>
+
+							</ul>
+						</div> -->
+
+
+						</li>
+
+
+
+            <li><a href="map.html">Map</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact Us</a></li>
+          </ul>
+        </div>
+
+      </div>
+    </header>
+
+		<div class="container">
+
+      <div class="cityName">
+        <h1>Washington, DC</h1>
+
+      </div>
+
+      <div class="row clearfix">
+        <a  class="tile" href="rinks/cabinjohn.html">
+            <h1>Cabin John</h1>
+            <p>SnS 8:30am - 10:00am</p>
+            <p>SnS 2:00pm - 3:00pm</p>
+        </a>
+        <a  class="tile" href="rinks/kettler.html">
+            <h1>Kettler Capitals Iceplex</h1>
+            <p>PU 12:20pm - 1:30pm</p>
+            <p>SnS 6:50pm - 8:10pm</p>
+        </a>
+        <a  class="tile" href="rinks/rockville.html">
+            <h1>Rockville Ice Arena</h1>
+            <p>SnS 1:15pm - 2:30pm</p>
+            <p>SnS 4:00pm - 5:10pm</p>
+        </a>
+      </div>
+      <div class="row clearfix">
+        <a  class="tile" href="rinks/reston.html">
+            <h1>Skatequest</h1>
+            <p>PU 9:00am - 10:20am</p>
+            <p>&nbsp</p>
```

Comment:
When you're encoding values like this they need to end in a semicolon - `&nbsp;`